SU-HSDO/suhumsci
# SHS-5636: Some Empty Headings in Banners still show up on SiteImprove

# READY FOR REVIEW

## Summary
Fix empty Headings in banners

## Need Review By (Date)
05/31

## Urgency
medium

## Steps to Test
1. Install the following sites and visit the following pages
    -  https://creativewriting.suhumsci.loc/undergraduate/creative-writing-minor
    - https://english.suhumsci.loc/academics/undergraduate-programs/english-major
    - https://mathematics.suhumsci.loc/academics/undergraduate-students
    - https://mathematics.suhumsci.loc/academics/undergraduate-students/math-major/prospective-math-majors
    - https://southasia.suhumsci.loc/about/make-gift
2. Confirm there are no empty headings in the banners
3. Go to this page on Colorful and Traditional or add a new page with all the different banners to confirm there aren't empty headings.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
